Introduce inclusive naming check and update existing language

aaronshurley opened the pull request:
We are following the Inclusive Naming Initiative's guidance as that is what the CNCF supports.

jbrunton commented:
Thanks for the PR, @aaronshurley!

It's currently failing because of a check that the generated code in `dist` is up to date (i.e. consistent with what happens when you run the `build` script), and as this includes source code from all dependencies, it's flagging a comment in a third party library (which appears to come from [node-tunnel](https://github.com/koichik/node-tunnel/blob/master/lib/tunnel.js#L214)).

I think ideally we should check the language in all the source files in this repo, but ignore `dist`: otherwise, even if I disable this PR check, we'll still have to fix this issue again every time the action is rebuilt and released.

It looks like `woke` supports [ignoring files](https://docs.getwoke.tech/ignore/). Any thoughts on adding an `ignore` input to the reusable workflow you created, to deal with this case? I'd be happy to submit a PR for this. Alternatively, I could implement a slightly modified version of the workflow for this repo. (I'm assuming these suggestions are compatible with CNCF guidance – if not, we can rethink.)

aaronshurley commented:
> Thanks for the PR, @aaronshurley!
> 
> It's currently failing because of a check that the generated code in `dist` is up to date (i.e. consistent with what happens when you run the `build` script), and as this includes source code from all dependencies, it's flagging a comment in a third party library (which appears to come from [node-tunnel](https://github.com/koichik/node-tunnel/blob/master/lib/tunnel.js#L214)).
> 
> I think ideally we should check the language in all the source files in this repo, but ignore `dist`: otherwise, even if I disable this PR check, we'll still have to fix this issue again every time the action is rebuilt and released.
> 
> It looks like `woke` supports [ignoring files](https://docs.getwoke.tech/ignore/). Any thoughts on adding an `ignore` input to the reusable workflow you created, to deal with this case? I'd be happy to submit a PR for this. Alternatively, I could implement a slightly modified version of the workflow for this repo. (I'm assuming these suggestions are compatible with CNCF guidance – if not, we can rethink.)

Thank you for the context! I'm happy to update this PR to ignore `dist` by adding it to a `.wokeignore` file. Feel free to let me know if you have any feedback on this approach. I'll make the update now.

aaronshurley committed:
Introduce inclusive naming check and update existing language

We are following the Inclusive Naming Initiative's guidance as that is what the
CNCF supports.

jbrunton commented:
Oh, excellent – I missed that there was a simple fix! LGTM.
